app/src/main/java/com/canvearth/canvearth/LoginActivity.java

```diff
@@ -45,26 +48,36 @@ public void onAuthStateChanged(@NonNull FirebaseAuth firebaseAuth) {
             }
         };
         mCallbackManager = CallbackManager.Factory.create();
-        LoginButton loginButton = findViewById(R.id.button_facebook_login);
+        final LoginButton loginButton = findViewById(R.id.button_facebook_login);
         loginButton.setReadPermissions("email", "public_profile");
-        loginButton.registerCallback(mCallbackManager, new FacebookCallback<LoginResult>() {
+//        loginButton.registerCallback(mCallbackManager, new FacebookCallback<LoginResult>() {
```

Comment:
Can you leave comment for this temporary comment-out?

Reply:
Done